sleap/gui/dataviews.py

```diff
@@ -347,6 +347,14 @@ def activateSelected(self, *args):
         if self.is_activatable:
             self.state[self.row_name] = self.getSelectedRowItem()
 
+            if self.row_name == "camera":
+                session = self.state["session"]
+                if (
+                    session is not None
+                    and session.get_video(self.getSelectedRowItem()) is not None
+                ):
+                    self.state["video"] = session.get_video(self.getSelectedRowItem())
```

Comment:
Good logic, but I dislike the placement inside `GENERICTableView` which is supposed to be... generic. A new class off of `GenericTableView`  where we override the `GenericTableView.activateSelected` is a good option. Or we could delegate this over to QtVideoPlayer where we do a similar thing for the `"unlinked_video"` `GuiState` key:
https://github.com/talmolab/sleap/blob/a32d68a1c116584c8950fde841310e77cc71ac99/sleap/gui/widgets/video.py#L278-L283